ext/CUDAExt/cudnn/conv.jl

```diff
@@ -0,0 +1,123 @@
+using NNlib: DenseConvDims
+import NNlib: conv!, ∇conv_filter!, ∇conv_data!, conv_bias_act!
```

Comment:
Yeah, there's plenty of cleanup that should happen on the NNlibCUDA side (I already deleted a couple of useless files). Didn't want to change too much though since that introduces extra testing/review burden.